app/src/main/java/com/checkin/app/checkin/Misc/BillHolder.kt

```diff
@@ -74,6 +74,6 @@ class BillHolder(itemView: View) {
             } else containerInvoiceDiscount.visibility = View.GONE
         }
         // Tip
-        tvInvoiceTip.text = Utils.formatCurrencyAmount(mContext, bill.tip)
+       // tvInvoiceTip.text = Utils.formatCurrencyAmount(mContext, bill.tip)
```

Comment:
This class is used by so many screens. Changing it requires some really good forethought. Why did you comment it out?